src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,19 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  categories: ['Under construction'],
+};
+
+const categoriesSlice = createSlice({
+  name: 'categories',
+  initialState,
+  reducers: {
+    checkStatus: (state) => {
+      console.log(state);
+      return state;
+    },
+  },
+});
```

Comment:
- [ ] HI good job setting up reducers, however please notice that as mentioned in the project requirements :point_down: 
![image](https://user-images.githubusercontent.com/97521557/233148814-fdffe683-3e0c-4966-8419-bf36bc785fc4.png)

please make sure the initial state is an empty array, and instead of logging the state to the console, you should return the string "Under construction", and default case is to return the initial state which will be an empty array, although I think Redux Toolkit automatically handles this by returning the current state by default, so you can just return "under construction" string :+1:

Reply:
This change has not been implemented well.